package.json

```diff
@@ -17,6 +17,7 @@
     "release:version": "changeset version && yarn install --immutable"
   },
   "peerDependencies": {
+    "@tanstack/react-query": "*",
```

Comment:
is there any risk in leaving `*`? like what if there is a major version issue?

Reply:
Oh I didn't notice that it added it with * when I did yarn install. will change